Library/Book.cpp

```diff
@@ -1,48 +1,29 @@
 #include "stdafx.h"
 #include "Book.h"
-
+#include <iostream>
 
 Book::Book()
 {
 }
 
-void Book::ViewBook()
-{
-	if (list.size() > 0) {
-		for (int i = 0; i < list.size(); i++)
-		{
-			std::cout << i+1 << ". " << list[i].book_title << std::endl;
-			std::cout << "   " << list[i].author_name << std::endl;
-			std::cout << "   " << list[i].year << std::endl;
-		}
-	}
-	else {
-		std::cout << "There is no books in the library" << std::endl;
-	}	
-}
-
-void Book::AddBook()
+void Book::NewBook()
```

Comment:
Just create constructor with these params. call to std::cin must be moved to a higher level of application